terraform/environments/mlra/application_variables.json

```diff
@@ -2,7 +2,12 @@
   "accounts": {
     "development": {
       "example_var": "dev-data",
-      "region": "eu-west-2"
+      "region": "eu-west-2",
+      "cloudwatch_stat": "Average",
+      "cloudwatch_view": "timeSeries",
+      "cloudwatch_type": "metric",
```

Comment:
"cloudwatch_stat": "Average",
      "cloudwatch_view": "timeSeries",
      "cloudwatch_type": "metric",
 
Are these attributes best placed here? Really, I think we'd want to retain this file for environment-specific inputs

Reply:
removed code from file